packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -71,7 +71,7 @@ export const createDefaultChatHandlers = memoizeOne(
         if (messageIterator === undefined) {
           // Lazy definition so that errors in the method call are reported correctly.
           // Also allows recovery via retries in case of transient errors.
-          messageIterator = chatThreadClient.listMessages();
+          messageIterator = chatThreadClient.listMessages({ maxPageSize: 50 });
```

Comment:
should we be allowing this to be configured by contoso and we can default to 50?

Reply:
We can talk about this later after GA - no matter how many messages our customer set, the chat thread behavior will still fetch new messages when you scrolling up, and fullfill the thread until the minimum scroll top gap is satisfied - it's more like an internal implementation for composite, so I would say we might not want to expose this api until our customers have a real requirement on it